auth/default_authorizer.cc

```diff
@@ -307,6 +314,47 @@ future<> default_authorizer::revoke_all(const resource& resource) const {
     });
 }
 
+future<> default_authorizer::revoke_all(const resource& resource) const {
+    if (legacy_mode(_qp)) {
+        co_return co_await revoke_all_legacy(resource);
+    }
+    static const sstring query = format("SELECT {} FROM {}.{} WHERE {} = ? ALLOW FILTERING",
+        ROLE_NAME,
+        _auth_ks_name,
+        PERMISSIONS_CF,
+        RESOURCE_NAME);
```

Comment:
Not sure it's a good idea for it to be `static`. Perhaps it's less controversial as this is only a non-legacy path, but it still raises an eyebrow.

Reply:
yes, I will do everything in one (future) commit, in those current commits I tried to not change many things at once.